src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheck.java

```diff
@@ -283,10 +287,7 @@ public void visitToken(DetailAST ast) {
         final int lineNo = ast.getLineNo();
         final TextBlock cmt =
             contents.getJavadocBefore(lineNo);
-        if (cmt == null) {
-            log(lineNo, MSG_MISSING_TAG, tag);
-        }
-        else {
+        if (cmt != null) {
             checkTag(lineNo, cmt.getText());
```

Comment:
> Now it is fixed properly. Weird message with null wont be reported as well.

Explain how weird message won't be reported when the log statement is in checkTag and there is nothing between here and there to skip logging if `tag` is `null`?

If we are skipping all violations when `tag` is `null` then we need to make it clear in documentation that tag must be specified.